Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:16.04
+
+ARG NEW_USER_UID
+ARG NEW_USER
+
+RUN apt-get -y update && \
+    apt-get install -y curl python3 python3-pip vim openssh-client \
+    git make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev \
+    openjdk-8-jdk
+
+RUN pip3 install tox virtualenvwrapper twine sphinx sphinx_rtd_theme wheel
+RUN pip3 install --pre github3.py
```

Comment:
I think this is not needed to run integration tests?

Reply:
No I thought we might keep it in case we give another try at #484 in the future with the same image. But I guess it makes sense to keep it clean for now.